src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/generated.py

```diff
@@ -126,3 +129,55 @@
 cli_storage_data_plane_command('storage file service-properties update', FileService.set_file_service_properties, factory)
 cli_storage_data_plane_command('storage file copy start', FileService.copy_file, factory)
 cli_storage_data_plane_command('storage file copy cancel', FileService.abort_copy_file, factory)
+
+# table commands
+factory = table_data_service_factory
+cli_storage_data_plane_command('storage table generate-sas', TableService.generate_table_shared_access_signature, factory)
+cli_storage_data_plane_command('storage table stats', TableService.get_table_service_stats, factory)
+cli_storage_data_plane_command('storage table service-properties show', TableService.get_table_service_properties, factory)
+cli_storage_data_plane_command('storage table service-properties update', TableService.set_table_service_properties, factory)
+cli_storage_data_plane_command('storage table list', TableService.list_tables, factory, simple_output_query='items[*].{Name:name}')
+cli_storage_data_plane_command('storage table create', TableService.create_table, factory)
+cli_storage_data_plane_command('storage table exists', TableService.exists, factory)
+cli_storage_data_plane_command('storage table delete', TableService.delete_table, factory)
+cli_storage_data_plane_command('storage table policy create', create_acl_policy, factory)
+cli_storage_data_plane_command('storage table policy delete', delete_acl_policy, factory)
+cli_storage_data_plane_command('storage table policy show', get_acl_policy, factory)
+cli_storage_data_plane_command('storage table policy list', list_acl_policies, factory, simple_output_query=transform_acl_list_output)
+cli_storage_data_plane_command('storage table policy update', set_acl_policy, factory)
+cli_storage_data_plane_command('storage table batch commit', TableService.commit_batch, factory)
+cli_storage_data_plane_command('storage table batch create', TableService.batch, factory)
+
+# table entity commands
+cli_storage_data_plane_command('storage entity query', TableService.query_entities, factory)
```

Comment:
Blob and File use the convention "storage blob" instead of "storage container blob" and "storage file" instead of "storage share file" because that is how the commands are in Xplat. 

While Xplat doesn't actually support messages or entities, I followed that naming convention so it would be consistent.

---
In reply to: [74452707](https://github.com/Azure/azure-cli/pull/681#discussion_r74452707) [](ancestors = 74452707)